acceptance/tests/options/config_file/debug.rb

```diff
@@ -22,8 +24,8 @@
       end
 
       step "debug output should print when config file is loaded" do
-        on(agent, facter("")) do
-          assert_match(/DEBUG/, stderr, "Expected DEBUG information in stderr")
+        on(agent, facter("")) do |facter_output|
```

Comment:
Out of curiosity, what does using the block argument do differently here? Is it just for readability or is the functionality different?